lib/data/file_manager/file_manager.dart

```diff
@@ -390,18 +396,36 @@ class FileManager {
         .map((FileSystemEntity entity) {
           String filePath = entity.path.substring(documentsDirectory.length);
 
-          // remove extension
-          if (filePath.endsWith(Editor.extension)) {
-            filePath = filePath.substring(
-                0, filePath.length - Editor.extension.length);
-          } else if (filePath.endsWith(Editor.extensionOldJson)) {
-            filePath = filePath.substring(
-                0, filePath.length - Editor.extensionOldJson.length);
-          }
-
+          //log.info('Listing file $filePath');
```

Comment:
nit: Seems like dead code, may with to remove.

Reply:
If removeExtension is true, then we should return only file names ending sbn2 or sbn. So it should be there I think

Reply:
Sorry I mean the log statement that is commented out.